build_tools/python/e2e_test_artifacts/cmake_generator/rule_generator.py

```diff
@@ -1,46 +0,0 @@
-## Copyright 2022 The IREE Authors
```

Comment:
Why is this being deleted? I don't see the corresponding new file. Did you forget to `git add` something?

Reply:
All logic here is moved to `generate_cmake_e2e_test_artifacts_suite.py`. There are some fragments in `generate_cmake_e2e_test_artifacts_suite.py` related to file handling and how to split the files. And `generate_cmake_e2e_test_artifacts_suite.py` is the only user of this module, so I think it's fair to remove this file.

Reply:
Ah, yeah I see there's less in here than I thought.